website/docs/final.md

```diff
@@ -0,0 +1,98 @@
+---
+id: final
+title: Final Methods, Classes, and Modules
+---
```

Comment:
This doesn't automatically show up in the sidebar. You'll need to edit `sidebars.json`.